packages/flutter_tools/lib/src/dart/summary.dart

```diff
@@ -74,3 +78,57 @@ void buildSkyEngineSdkSummary(
   String outputPath = pathos.join(skyEnginePath, outBundleName);
   new io.File(outputPath).writeAsBytesSync(output.sum);
 }
+
+Future<Null> buildUnlinkedForPackages(String flutterPath) async {
+  PhysicalResourceProvider provider = PhysicalResourceProvider.INSTANCE;
+  PubSummaryManager manager =
+      new PubSummaryManager(provider, '__unlinked_${io.pid}.ds');
+
+  Folder flutterFolder = provider.getFolder(flutterPath);
+
+  //
+  // Build in packages/.
+  //
+  Folder packagesFolder = flutterFolder.getChildAssumingFolder('packages');
+  await _buildUnlinkedForPackage(
```

Comment:
Honestly, I did not wrap it at all, I just thought that formatter does the right thing.
I will keep eye on this in the future.
This code was replaced in a next commit.